wIndex/Diagnostics.md

```diff
@@ -42,12 +48,12 @@ The Public tools involves Logging, Cache, and Bug Fixes.
 
 | Setting | Description |
 |----|----|
-| Clear External Login Cookies | Click Execute to clear external login cookies. This will delete any saved form data in the login window as well.<br><br>If you log in with an external auth provider you may be offered an option to remember your login. This may create a security risk in some cases. You can use this feature to clear the data stored for your login so other users can not log in with your cached auth cookie.|
-| Toggle Verbose Logging | Click Execute Selected Action to toggle Verbose Logging. This adds very detailed logs on hundreds of application events for detailed analysis by support folks. |
-| Toggle Memory Logging | Click Execute Selected Action to toggle Verbose Logging. This adds very detailed logs on hundreds of application events for detailed analysis by support folks.<br><br>No input required.  You may change the default 3000 millisecond delay by replacing this text with the number of milliseconds required.  The setting must be between 1000 and 30000 milliseconds. |
-| Toggle DataCell Audit Mode | Click 'Execute Selected Action' to Enable or Disable the Formula Auditing Mode for DataCell calculations.  This will provide you exact cell address data for errors noted when you re-run the reports. |
-| Reset Settings Cache | This will reset the user setting cache so the system will be forced to repull from the central database. |
-| Open User Folders | Click Execute to open the folders for user application data, client shared data and report file cache. |
+| [Clear External Login Cookies](/wAbout/Logging-In-Enterprise.html#clearing-login-cookies) | Click Execute to clear external login cookies. This will delete any saved form data in the login window as well.<br><br>If you log in with an external auth provider you may be offered an option to remember your login. This may create a security risk in some cases. You can use this feature to clear the data stored for your login so other users can not log in with your cached auth cookie.|
+| [Toggle Verbose Logging](/wTroubleshoot/Verbose-Logging.html) | Click Execute Selected Action to toggle Verbose Logging. This adds very detailed logs on hundreds of application events for detailed analysis by support folks. |
+| [Toggle Memory Logging](/wTroubleshoot/RAM-Monitoring.html) | Click Execute Selected Action to toggle Verbose Logging. This adds very detailed logs on hundreds of application events for detailed analysis by support folks.<br><br>No input required.  You may change the default 3000 millisecond delay by replacing this text with the number of milliseconds required.  The setting must be between 1000 and 30000 milliseconds. |
```

Comment:
same here

Reply:
"folks" is what is displayed in the diagnostic form so this is just a direct iteration of that